### PR TITLE
Updated code blocks

### PR DESCRIPTION
**Pull request by hankandre**
I've updated all the code blocks that I could find where the indentation needed to be adjusted down from 4 to 2 spaces.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -637,23 +637,23 @@ There's no built-in mechanism for default inputs, because of this we assign them
...
-        if (typeof bindings.displayPurchased.currentValue === 'undefined') {
-            this.displayPurchased = true;
-        }
-        if (typeof bindings.displayAvailable.currentValue === 'undefined') {
-            this.displayAvailable = true;
-        }
+  $onChanges(bindings) {
+    if (typeof bindings.displayPurchased.currentValue === 'undefined') {
+      this.displayPurchased = true;
+
```
Comment (feimosi) [inline README.md]: `}` is missing here in the `if`

File: README.md [hunk 2]
```diff
@@ -1704,37 +1704,36 @@ You can use the `v-model` directive to create **two-way data bindings** on form
...
+  <label>
+    Email:
+    <input type="text" ng-model="$ctrl.email" />
+  </label>
+
+  <label>
+    E-mail:
+    <input type="email" ng-model="$ctrl.password" />
+  </label>
+  <input type="submit" ng-click="$ctrl.submit()" value="Save" />
```
Comment (feimosi) [inline README.md:1720]: Please add an empty line before `<input`

**Review by feimosi — CHANGES_REQUESTED** [review]
Thanks! Take a look at the comments.

**Comment by feimosi** [comment]
Alright I've fixed it myself.

**Comment by hankandre** [comment]
Thanks. Sorry I didn't get to it for you.

**Comment by feimosi** [comment]
Yeah, no problem!

**Pull request merged**